pkg/reconciliation/reconcile_racks.go

```diff
@@ -5,11 +5,11 @@ package reconciliation
 
 import (
 	"fmt"
+	"net"
 	"reflect"
 	"sort"
 	"strings"
 	"time"
-	"net"
```

Comment:
Unrelated, not sure why this happened (Go version maybe?)
I can revert if necessary.

Reply:
Yeah, go fmt in 1.19 has some changes.